src/rgw/rgw_common.h

```diff
@@ -1132,10 +1127,6 @@ struct RGWBucketInfo {
   //   - value of 0 indicates there is no sharding (this is by default
   //     before this feature is implemented).
   //   - value of UINT32_T::MAX indicates this is a blind bucket.
-  uint32_t num_shards{0};
```

Comment:
The variable was removed, but the comments immediately before describing the variable were left behind, making reading the code confusing.

Reply:
ack. will address in the currently open pr [35175](https://github.com/ceph/ceph/pull/35175)